lcls-twincat-common-components/Library/Devices/PPM/FB_PPM_PowerMeter.TcPOU

```diff
@@ -28,61 +41,145 @@ VAR
     fVoltageBuffer: ARRAY[1..1000] OF LREAL;
 
     {attribute 'pytmc' := '
-        pv: CALIB
+        pv: STC
         io: input
     '}
-    fCalibBase: LREAL;
+    fbTempSensor: FB_CC_TempSensor;
 
+    // Used to get beam mode and rate
     {attribute 'pytmc' := '
-        pv: CALIB_BUFFER
-        io: input
+        pv: AccTypeFromEpics
+        link: TPR:FEES:GD02:0:MODE.RVAL
+    '}
+    uAccelerator: UINT;
```

Comment:
This link for `uAccelerator` is a bit suspicious, it has `FEES` in it which implies it might only be correct for the soft-xray line and there may be an alternate PV for the accelerator type on the hard x-ray line